### DISCUSSION TITLE
Want to increase diameter and change selection method of selected points of the cell over which a stretchingforce is applied

### DISCUSSION THREAD

**Discussion by rishabhbhatt233** [UvaCsl/HemoCell#116] [General discussion]
Hello 
    I am new to 'Hemocell' and trying to use "stretchCell" example, how can I increase the stretching region.I have done <prctForced> to  1 but I want to increase the diameter of region from where vertices are being select. Also in this one of the vertex of RBC is stretched larger but I want to stretch RBC as the two beads are attached to it and that whole part of the surface get force of stretch(stretched vertices are part of spherical region) as in figure (I have attached) is shown. How can I do this? Also in "stretchCell.cpp", what "unsigned n_forced_lsps = 1 + 6;" represents and used for?

Thank you
![Screenshot from 2023-12-26 23-58-34](https://github.com/UvaCsl/HemoCell/assets/153628181/2f7d93b1-547c-45b4-8fa3-db6b846bf59e)

**Comment by mojtabaAmiraslanpour** [comment]
You can set prctForced in the config.xml to set the percent of the nodes selected:

<prctForced>0.05</prctForced>

The default is 5%.

**Reply by rishabhbhatt233** [reply]
Thank you for your response.
As I have tried it but I don't know why only one particle is stretched 
![Screenshot from 2024-01-09 17-48-27](https://github.com/UvaCsl/HemoCell/assets/153628181/1027a1a2-8931-417f-8e85-d00b93f3e626)

I want to stretch RBC from all the selected particles. Can I do this? 

Also I have tried "Validation" for minimum triangle of 600 and then 6000 and higher but I got the distortion in output 
I have tried to change stiffness of membrane by changing max angle but still I got distortion and cannot get validation
![validation (1)](https://github.com/UvaCsl/HemoCell/assets/153628181/0e135e2d-17b6-451c-9582-08528da111ff)
